src/main/java/racingcar/vo/CarList.java

```diff
@@ -0,0 +1,68 @@
+package racingcar.vo;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.model.Car;
+
+public class CarList {
+    private static final String STRING_WINNER_DELIMITER = ", ";
+    private final List<Car> carList;
+
+    public CarList(final List<Car> carList) {
+        this.carList = carList;
+    }
+
+    public List<Car> getCarList() {
+        return carList.stream()
+                .collect(Collectors.toUnmodifiableList());
+    }
```

Comment:
`toList()`를 사용하시면 동일하게 불변 컬렉션을 얻을 수 있어요!
```suggestion
    public List<Car> getCarList() {
        return carList.stream()
                .toList();
    }
```
![image](https://github.com/woowacourse-precourse/java-racingcar-6/assets/73726591/05e2ab70-89c6-4938-8993-80676c0633ad)